apps/AEPSampleApp/README.md

```diff
@@ -1,32 +1,79 @@
 # AEPSampleApp
 
-> Note: Commands are assuming you're in the root directory of the repository.
+## Prerequisites
 
-> Note: If you are new to React Native we suggest you follow the [React Native Getting Started](<https://facebook.github.io/react-native/docs/getting-started.html>) page before continuing.
+ * If you are new to React Native we suggest you follow the [React Native Setting up development environment](https://reactnative.dev/docs/environment-setup) React Native CLI Quickstart page before continuing.
+ * Cocoapods version >= 1.11.3
+ * Node version >= 18
 
-How to run the sample app:
+## How to run the sample app:
 
 ### Add your App Id:
+
 In `ios/**/AppDelegate.mm`, find the call to `configureWithAppId` and add your app id.
 
 In `android/**/MainApplication.java`, find the call to `configureWithAppId` and add your app id.
 
```

Comment:
- can we use the full path for these files from the root directory, e.g. apps/AEPSampleApp/ios/AEPSampleApp/AppDelegate.mm
- can we add simple instructions on where to find the app id?

Reply:
I have a note section right below to point to the user how to setup mobile property.  Is that good enough?  We should assume them already have the knowledge to setup the app id and schema, etc before going to this sample app.

App id is configured in a mobile property in Data Collection UI, in Tags. Refer to [configure with App ID per environment](https://developer.adobe.com/client-sdks/home/base/mobile-core/configuration/#configure-with-app-id-per-environment) page for more information.